apps/server/package.json

```diff
@@ -26,8 +26,10 @@
     "@babel/eslint-parser": "^7.18.2",
     "@capacitor-community/http": "^1.4.1",
     "@elastic/elasticsearch": "^8.13.1",
+    "@elastic/elasticsearch-mock": "^2.0.0",
```

Comment:
This lib is used in normal code, so needed to be moved here. Docker build skips devDeps